setup.py

```diff
@@ -36,7 +36,8 @@
     'codecov~=2.0',
     'future',
     'six',
-    'hypothesis'
+    'hypothesis',
+    'crccheck'
```

Comment:
This needs to be put in `install_requires` (found bottom of the file), since `test_require` is only installed for the tests.

Reply:
Or better put it as an extra like the neovi interface.

Reply:
setup.py has been updated with crccheck as an option in extras_require.  The option is added with the name 'crccheck' so its not too specific to one interface; also because usb-can-analyzer is a bit wordy.  

In addition a try - except on importing the module was added to point users in the right direction.  It will likely still crash out without crccheck installed if using usb-can-analyser.  Not sure if there's a better way?